libpysal/graph/base.py

```diff
@@ -403,6 +403,56 @@ def build_contiguity(cls, geometry, rook=True, by_perimeter=False, strict=False)
         -------
         Graph
             libpysal.graph.Graph encoding contiguity weights
+
+        Example
+        -------
```

Comment:
```suggestion
        Examples
        --------
```
Needs to be plural to be understood by sphinx (docs builder).